data.sql

```diff
@@ -37,5 +37,33 @@ UPDATE animals SET owner_id = 4 WHERE name IN ('Charmander', 'Squirtle', 'Blosso
 UPDATE animals SET owner_id = 5 WHERE name IN ('Angemon', 'Boarmon');
 COMMIT;
 
+INSERT INTO vets (name, age, date_of_graduation) VALUES ('William Tatcher', 45, '2000-7-23');
+INSERT INTO vets (name, age, date_of_graduation) VALUES ('Maisy Smith', 26, '2019-1-17');
+INSERT INTO vets (name, age, date_of_graduation) VALUES ('Stephanie Mendez', 64, '1981-3-4');
+INSERT INTO vets (name, age, date_of_graduation) VALUES ('Jack Harkness', 38, '2008-5-8');
```

Comment:
- [ ] Kindly fix the graduation month for Jack Harkness.
  `Vet Jack Harkness is 38 years old and graduated Jun 8th, 2008.`

Reply:
@mahbubul14 Thank you